financial-services/1.10/modules/ROOT/pages/rpa-setup-guide.adoc

```diff
@@ -0,0 +1,167 @@
++
+image::rpa-setup-keystrokes-app-element-wizard.png[RPA Builder Keystrokes to App Element Wizard]
+
+. Re-identify the *Login* button to click on:
++
+image::rpa-setup-click-app-element-wizard.png[RPA Builder Click App Element Wizard]
+
+. Make sure you wait for the login process to complete by choosing a pattern that will disappear after completion:
++
+image::rpa-setup-choose-pattern.png[RPA Builder Image Search Wizard choose pattern]
+
+. Adapt the other Action Steps of the same types in a similar way. Or, add or delete Action Steps based on what is needed to log in to your AMS.
+
+=== Activity 2: Find a Record in the AMS
+
+The second activity finds a record in the AMS using a lookup code.
+
+. Enter the lookup code, which was provided as an Activity Parameter, as locate criterion to search for the account data set.
+. Click on the result entry to open the account data and wait until it is fully loaded.
+
+image::rpa-setup-lookup-code.png[RPA Builder lookup code]
+
+Change the following Action Steps:
+
+. Adapt the first Action Steps as you did in the previous Activity.
+. Adapt the scan area in the AI OCR Action Step in which RPA Bot searches for the record entry with the lookup code:
++
+image::rpa-setup-ai-ocr-wizard.png[RPA Builder AI OCR screen-based Wizard]
+
+. Adapt the rest of the Action Steps as you did in the previous Activity.
+
+=== Activity 3: Copy the data from the AMS to 
+
+The third activity retrieves data and exports the data from the AMS to Salesforce Financial Services Cloud.
+
+. Retrieve account data.
+. Retrieve the policy data of the account.
+. Export the data in one of the following ways:
+.. CSV file
+.. REST call
+
+image::rpa-setup-export-data.png[RPA Builder export data]
+
+Change the following Action Steps:
+
+. To retrieve the data, re-identify the fields from which to read in all the Read Text from App Element Action Steps and add or delete them as needed.
+
+image::rpa-setup-read-text-app-element-wizard.png[RPA Builder Read Text from App Element Wizard]
+
+It is sometimes necessary to click into a field before reading the text, and then to use a keystroke to get out of the field afterward.
+
+image::rpa-setup-read-text-tip.png[RPA Builder tip for getting in and out of fields]
+
+If you search for text in a special format, you can use Optical Character Recognition in combination with regular expressions.
+
+image::rpa-setup-optical-character-recognition.png[How to use optical character recognition in RPA Builder]
+
+[start=2]
+. Ensure all retrieved data is pinned to the Combine Strings Action Steps and adjust if necessary. You can also link more Activity Parameters. 
++
+image::rpa-setup-combine-strings-wizard.png[RPA Builder Combine Strings Wizard]
+
+. Add the string to a CSV file via the `Pin Variable`. You can adjust the properties as needed. The CSV file must be in the format specified.
++
```

Comment:
More info on the CSV format might be found here:

- https://help.salesforce.com/s/articleView?id=000381876&type=1
- https://developer.salesforce.com/docs/atlas.en-us.api_asynch.meta/api_asynch/datafiles_field_names.htm
- https://developer.salesforce.com/docs/atlas.en-us.248.0.object_reference.meta/object_reference/sforce_api_objects_list.htm
- https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_account.htm

Reply:
see our project chat